beyond-essence/inventory/buttons.md

```diff
@@ -1,6 +1,86 @@
 ---
 layout: beyond-essence
+header_image: private/main.jpg
+title: Buttons
+background_image: index.jpg
 ---
 
-Buttons are basic elements in user interfaces.
-It is important to design an effective button that will make the user take action.
+Buttons are interactive elements and belong to the action controls.
+They allow the user to directly communicate with the user interface.
+
+## Use cases
+
+With buttons we allow the user to take an action upon click or touch.
+Buttons appear at various places throughout the UI, such as:
+
+* Cards
+* Dialogs
+* Settings layers
+* Top bar
+* Bottom bar
+* Themes
+* Empty states
+
+## Structure
+
+Buttons are characterized by a certain geometric shape or border.
+Their color clearly stands out from the surrounding color.
+The UI label appears on the button, and clearly explains what the button will do.
+In exeptional cases or where the situation requires it, buttons appear with an icon instead of a UI label.
+
+## Position
+
+Buttons are positioned where users can easily find them or expect to see them.
+The position of the buttons should be intuitive for the user and reflect a natural conversation between the user and the software.
+The order and position of buttons is especially important when they come in pairs, such as `Cancel` and `Save`.
+If the user has the choice between two buttons, one distinguishes between primary and secondary buttons.
+The primary button is the one that completes the user's task, whereas the secondary button has less importance and is rather counterproductive for task completion.
+
+The primary button is always located on the right side, the secondary button on the left side:
+
+{% image_custom image="/assets/img/pages/essence/buttons-primary-secondary.png" width="20" lightbox %}
+
```

Comment:
A remark:
Normally primary and secondary buttons are grouped as a pair (that´s true) but there are also exceptions, e.g. in the variation product context.
Here they are positioned separately from each other- because next to the primary button an additional info is placed. 

<img width="688" alt="Bildschirmfoto 2019-08-12 um 09 11 35" src="https://user-images.githubusercontent.com/6104802/62849923-b4e3c400-bce1-11e9-8708-6f0ddac4ed31.png">

According to the example `In the bottom bar`.
Do we wanna add a rule here? If yes then we could also mention that the alignment is the following: 
The delete button is then placed on the left side while primary and secondary button are positioned as a pair on the right side.